controller/contour/suite_test.go

```diff
@@ -37,9 +42,25 @@ import (
 // These tests use Ginkgo (BDD-style Go testing framework). Refer to
 // http://onsi.github.io/ginkgo/ to learn more about Ginkgo.
 
-var cfg *rest.Config
-var k8sClient client.Client
-var testEnv *envtest.Environment
+// Define utility constants for object names, testing timeouts/durations intervals, etc.
+const (
+	contourName       = "test-contour"
+	operatorNamespace = "test-contour-operator"
+	defaultNamespace  = "projectcontour"
+	defaultRemoveNs   = false
+	defaultReplicas   = int32(2)
+
+	timeout  = time.Second * 10
+	interval = time.Millisecond * 250
+)
+
+var (
+	cfg       *rest.Config
+	k8sClient client.Client
+	testEnv   *envtest.Environment
+	ctx       = context.Background()
```

Comment:
Just make a new context where you need one?

Reply:
ack